creator-node/src/routes/users.js

```diff
@@ -28,8 +28,16 @@ module.exports = function (app) {
     return successResponse()
   }))
 
+  app.get('/users/login/challenge', handleResponse(async (req, res, next) => {
+    const walletPublicKey = req.query.walletPublicKey
+    const redisClient = req.app.get('redisClient')
+    const challenge = (new Date()).getTime()
+    redisClient.set(walletPublicKey, challenge)
```

Comment:
@vicky-g in addition to the await, lets keep make a unique redis key to look this info up. Lets do like `userLoginChallenge:<wallet>:<randomBytes`